pages/common/codespell.md

```diff
@@ -0,0 +1,36 @@
+# codespell
+
+> Spellchecker for source code.
+> More information: <https://github.com/codespell-project/codespell>.
+
+- Check for typos in all text files in the current directory, recursively:
+
+`codespell`
+
+- Correct all typos found in-place:
+
+`codespell --write-changes`
+
+- Skip files with names that match the specified pattern (accepts a comma-separated list of globs):
```

Comment:
Is there maybe another word for `glob`? I didn't know what a glob is until now and I think there are many others. (That's just my experience, maybe it is more common than I think). Maybe  `accepts a comma-seperated list of patterns using wildcards`?

Reply:
Oh, yes, definitely. I didn't know that 🙂.  How about `wildcard patterns` instead of `patterns using wildcards`?

Reply:
I'd prefer `patterns using wildcards`

Reply:
```suggestion
- Skip files with names that match the specified pattern (accepts a comma-separated list of patterns using wildcards):
```